root/etc/e-smith/events/actions/nethserver-firewall-base-rmmod-SipH323-modules

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Copyright (C) 2020 Nethesis S.r.l.
+# http://www.nethesis.it - support@nethesis.it
+# 
+# This script is part of NethServer.
+# 
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+# 
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+#
+# Disable SIP ALG for asterisk
+#
+SIPALG=$(/usr/sbin/e-smith/config getprop firewall SipAlg)
+
+if [[ $SIPALG != 'enabled' ]];then
+    /usr/sbin/modprobe -r nf_nat_h323
+    /usr/sbin/modprobe -r nf_conntrack_h323
+    /usr/sbin/modprobe -r nf_nat_sip
+    /usr/sbin/modprobe -r nf_conntrack_sip
```

Comment:
I'm pretty sure this will not work if there is any connection using such helper.

A better approach could be create a new event which reset the conntrack and then unload the modules.
Before implementing it, you should replicate the scenario and see if erasing the conntrack allows the unloading of kernel modules.

Reply:
done